.eslintrc

```diff
@@ -4,7 +4,11 @@
         "sourceType": "module",
         "ecmaFeatures": {}
     },
-    "extends": ["standard"],
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "standard"
```

Comment:
trailing commas need, not because IE8
https://time2hack.com/are-you-using-trailing-commas-in-your-javascript/
https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8